src/App.tsx

```diff
@@ -1,84 +1,18 @@
 /**
  * @file Contains the App top level component.
  */
-import Box from '@mui/material/Box';
-import React, { useState } from 'react';
-import Button from '@mui/material/Button';
-import DragAndDropzone from './components/DragAndDropzone';
+
+import React from 'react';
+import Menu from './components/Menu';
```

Comment:
Code does not run because Menu should be lower case 
![image](https://user-images.githubusercontent.com/95719901/229809212-78b05089-00f4-429a-81f3-35300cf90560.png)

![image](https://user-images.githubusercontent.com/95719901/229809311-9cfa3fac-4e7c-445c-a55f-42de43aecf26.png)

Reply:
The files have been renamed on GitHub as well